### DISCUSSION TITLE
Quality Time on RHEL 8

### DISCUSSION THREAD

**Discussion by makeitstatic** [ICTU/quality-time#6801] [Q&A]
Hi,

I am trying to run QT on SE-Linux enabled RHEL 8 with Podman in rootless configuration.
The networking seems to give failures, perhaps because of how the users and networking is set up in the QT components' dockerfiles.

Has anyone else been successful with running QT in a RHEL8 rootless setup?

Kindly,
makeitstatic

**Comment by fniessink** [comment]
Sorry, I have no experience with that specific setup.
